README.md

```diff
@@ -1 +1,170 @@
-# conference
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<h3><b>Conference website README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [portfolio Project] <a name="about-project"></a>
+
+**[portfolio Project]** is a simple HTML, CSS and Javascript webpage design from module 1 of Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+    <li><a href="w3school.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add html webpage]**
+- **[Add readme.md files]**
+- **[Add MIT license file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="https://https://peteronyam.github.io/conference/"></a>
+
+- [Live Demo Link](https://https://peteronyam.github.io/conference/)
```

Comment:
- [ ] Hi, I see that you tried to deploy your project, That is cool. What I noticed is the link to the deployment of your website is broken. I would suggest that you work on the deployment again and deploy it properly. Deployment is one of the project requirements, also it helps the recruiter to easily evaluate your skills and then give you a Job 🥇